docs/book/src/getting-started/upgrades.md

```diff
@@ -0,0 +1,38 @@
+# Upgrades
+
+This page includes instructions for upgrading the driver to the latest version.
+
+```bash
+helm upgrade csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driver
```

Comment:
```suggestion
helm upgrade csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driver --namespace kube-system
```

Reply:
If the previous installation is in a different namespace, then this command will fail. We could use `NAMESPACE` var here and recommend replacing with the namespace where there current release is in.

Reply:
Updated, let me know if this is what you meant by `NAMESPACE`